gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat Nov 03 11:02:51 CET 2018
+##Sat Nov 03 11:02:51 CET 2018
```

Comment:
This file shouldn't be changed.

Reply:
Hey @neslihanturan in response to your suggestion of returning a null instead of an empty object

I had already tried this before and on returning null the observable where this value is returned generates another exception since it can't handle a null response